### DISCUSSION TITLE
listing wallet in walletconnect explorer

### DISCUSSION THREAD

**Discussion by 0xweb3builder** [WalletConnect/walletconnect-monorepo#2912] [Developer Support]
`6690e7702b9dcc599d2bfefc3243a357` is our project id.
We migrated our wallet from v1.0 to v2.0
Our app was listed before, but now it is not shown in walletconnect app list.
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/32242962/b136c242-ec4a-476e-83d6-00ad7d0a3d12)
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/32242962/ba0f922f-7f2e-4e7b-a04c-eaeb7a2efc6c)

Please help us with app listing.

**Comment by crypblizz8** [comment]
Investigating for you now!

**Reply by JeroldG** [reply]
Hi all,

Your listing is approved and published on Explorer here: https://walletconnect.com/explorer/chain

I did notice that you are missing mobile Deeplink and a universal link which needs to be correctly entered so that your users can find your wallet when searching on Web3Modal. Please update these on your explorer submission.

Link: https://cloud.walletconnect.com/admin/explorer-listing?uuid=2b9f2f0b-3d72-453e-9409-609f296f4d54
